students/noahjones

```diff
@@ -0,0 +1,351 @@
+<!DOCTYPE html>
+
+<head>
+    <style>
+        /* google fonts */
+        
+        @import url(https://fonts.googleapis.com/css2?family=Arvo:wght@700&family=Indie+Flower&family=Playfair+Display:wght@600&family=Righteous&family=Rowdies&display=swap);
+        * {
+            transition: all 0.3s ease;
+        }
+        
+        body {
+            font-family: "helvetica", sans-serif;
+            font-weight: bold;
+            font-size: 100%;
+            color: white;
+            background: black;
+        }
+        
+        .wrapper {
+            width: 10%;
+            margin: auto;
+            height: 10vh;
+            display: flex;
+            justify-content: center;
+            align-items: center;
+        }
+        
+        .m:hover,
+        .e:hover,
+        .g:hover,
+        .a:hover,
+        .n:hover,
+        .ex:hover {
+            padding: 0 5px;
+            cursor: none;
+        }
+        
+        .m {
+            color: white;
+            font-family: "Righteous", cursive;
+        }
+        
+        .e {
+            color: white;
+            font-family: "Indie Flower", cursive;
+        }
+        
+        .e:hover {
+            color: yellow;
+        }
+        
+        .g {
+            color: white;
+            font-family: "Arvo", serif;
+        }
+        
+        .a {
+            color: white;
+            font-family: "Playfair Display", serif;
+        }
+        
+        .n {
+            color: white;
+            font-family: "Rowdies", cursive;
+        }
+        
+        .m:hover {
+            color: black;
+            background-color: white;
+            font-size: 1rem;
+        }
+        
+        .e:hover {
+            color: black;
+            background-color: white;
+            font-size: 1rem;
+        }
+        
+        .g:hover {
+            color: black;
+            background-color: white;
+            font-size: 1rem;
+        }
+        
+        .a:hover {
+            color: black;
+            background-color: white;
+            font-size: 1rem;
+        }
+        
+        .n:hover {
+            color: black;
+            background-color: white;
+            font-size: 1rem;
+        }
+        
+        .ex {
+            color: white;
+        }
+        
+        .ex:hover {
+            color: black;
+            background-color: white;
+            font-size: 1rem;
+        }
+    </style>
+    <style>
```

Comment:
❓ Why are there two `<style>` tags, by chance? Would you be able to achieve the same results with one?

Reply:
I think I did this by mistake